src/pages/indexStyle.module.scss

```diff
@@ -0,0 +1,3 @@
+.sectionWrapper {
+  margin: 20px 0 60px;
```

Comment:
I suppose don't use a separate file for one rule and rewrite this in inline styles